uuid_test.go

```diff
@@ -634,8 +634,10 @@ func TestValidate(t *testing.T) {
 	}
 }
 
-var asString = "f47ac10b-58cc-0372-8567-0e02b2c3d479"
-var asBytes = []byte(asString)
+var (
```

Comment:
This is unrelated to the PR

Reply:
Solved in [54f9572](https://github.com/google/uuid/pull/166/commits/54f95728c20a700436b4f3a8e699a99235748175)